openfisca_france/parameters/prestations/minima_sociaux/aspa/montant_annuel_couple.yaml

```
@@ -36,3 +36,5 @@ values:
   2020-01-01:
     value: 16826.64
     reference: https://www.legifrance.gouv.fr/eli/decret/2018/3/30/SSAS1805864D/jo/article_1
+  2021-01-01:
+    value: 16893.94
```

Pas de décret comme en 2020 ? 🤔 Circulaire CNAV (absente des [barèmes IPP](https://www.ipp.eu/baremes-ipp/prestations-sociales/3/minimum_vieillesse_droits_non_contributifs_de_retraite/aspa/) à ce stade) : 
```suggestion
    value: 16893.94
    reference: https://www.legislation.cnav.fr/Documents/circulaire_cnav_2021_01_11012021.pdf
```